providers/register-an-api.md

```diff
@@ -0,0 +1,88 @@
+# How to register an API user for a customer
+
+This document describes how a consumer can gain access to an API exposed on the Logic Marketplace.
+
+## 1. Access request from the customer
+
+In order to get access to Logic, the consumer needs to be able to access the [Logic Console](console). Today there are 2 different ways to achieve this - setting up a federation with Logic Identity and the consumers own identity store (company or municipal AD/AAD) or requesting to have a user created in the Logic Console user store. Fro both, the consumer needs to contact [support](mailto:logic-admin@kmd.dk) with the following information.
+
+- Company name
+- CVR number of the company
+- Contact information:
+  - Full name of the contact person
+  - Email address
+  - Phone number
+  - Company address
+
+## 1. Create a Logic Subscription
+
+When the consumer/partner is setup with access to the console the consumer will now have to setup a Logic subscription through the [Logic Console](console)
+  
+1. Go to [Logic Console](console)
+2. Choose `Subscriptions` from the menu and then click on `Add Subscription`
+3. Enter a `Subscription Name` of the partners name. A `Starter` subscription is fine. Click on `Create`
```

Comment:
You can remove the reference to Starter now

Reply:
Good point. Thanks